src/components/InteractiveList/InteractiveListInt.part.tsx

```diff
@@ -428,7 +441,8 @@ export class InteractiveListInt extends React.PureComponent<InteractiveListProps
 
   private createItem(key: string, index: number, content: React.ReactChild) {
     const { value, selected } = this.state;
-    const { disabled, showTick = false, multiple = false, condensed = false } = this.props;
+    const { disabled, showTick = false, multiple = false, condensed = false, disabledItems } = this.props;
```

Comment:
You can initialize disabled items by default with an array and avoid checking, like it's being done in all other props.

`disabledItems = []`